devtools/conda-envs/user_env.yaml

```
@@ -0,0 +1,27 @@
+name: test
```

We can take advantage of the name in this file. We could say 

```suggestion
name: opencadd
```
and then we can skip the `-n  <name>` flag when installing. (Or the name you prefer).